cluster/windows-demo.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: kubevirt.io/v1alpha1
```

Comment:
@fabiand managed to boot a windows server 2012 r2 with that example. @michalskrivanek glad for any comments regarding to the feature and clock/timer configuration in the example?